lib/utils.js

```diff
@@ -1033,6 +1033,16 @@ function whilst(condition, fn, callback) {
   }
 }
 
+/**
+ *
+ * @param {String} entity
+ * @param {String} name
+ * @returns
+ */
+function markAsNotSupported(entity, name) {
```

Comment:
Maybe consider moving this to ``new-utils.js`` like in [duration PR](https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/blob/f7fd151c5e44dc836a17a15030e37dddcfd54b3a/lib/new-utils.js), to differentiate between new and old code? Later we could merge those two files.

Reply:
I like the idea, it's going to bring more clarity to the code. I think _later_ in this context is going to be quite distant future :)